mlserver/batch_processing.py

```diff
@@ -88,8 +88,12 @@ def json_to_triton(
     inference_request = InferenceRequest.parse_obj(orjson.loads(request_json))
     for request_input in inference_request.inputs or []:
         new_input = httpclient.InferInput(
-            request_input.name, request_input.shape, request_input.datatype
+            request_input.name,
+            request_input.shape,
+            request_input.datatype,
         )
+        if request_input.parameters is not None:
+            new_input._parameters.update(request_input.parameters.dict())
```

Comment:
Wondering about this change and exact motivation behind. It is true that currently there is no option to set parameters on the inputs and request as a whole but I am worried that modifying triton's internal parameters (as indicated by `_`) could lead to unexpected breaking changes in the future.